### DISCUSSION TITLE
Discern spoken word versus music

### DISCUSSION THREAD

**Discussion by ifeatu** [PaddlePaddle/PaddleSpeech#2837] [Ideas]
Hello!  This is a tremendous project!  I'm glad I stumbled on it. 

I have a unique challenge: I am trying to determine whether a track has only spoken word or is music.  I'm looking for something binary.  Yes it has spoken word or no it does not.  I need to filter through hundreds of thousands of tracks with either spoken word or music.  I'd rather not listen to every one.  Thanks for any suggestions any one might provide here.

**Comment by yt605155624** [comment]
I don't have any idea about how to determine "whether a track has only spoken word or is music", but I have some idea about "how to split spoken word and background music in a **music with human voice**"
1. split human voice and music with band-pass filter and band-stop filter (You should have some knowledge of digital signal processing)
2. try https://github.com/deezer/spleeter

Hope this infomation will help you
